examples/ex3.cpp

```diff
@@ -53,6 +60,8 @@ int main(int argc, char *argv[])
    const char *mesh_file = "../data/beam-tet.mesh";
    int order = 1;
    bool static_cond = false;
+   bool pa = false;
+   const char *device_config = "cpu";
    bool visualization = 1;
```

Comment:
It is really awesome to see Device support in more examples, thank @dylan-copeland !